quartz-core/src/main/java/org/quartz/core/QuartzScheduler.java

```diff
@@ -1589,6 +1589,19 @@ public TriggerState getTriggerState(TriggerKey triggerKey) throws SchedulerExcep
 
     /**
      * <p>
+     * Reset the current state of the identified <code>{@link Trigger}</code>.
```

Comment:
I agree - I wonder if the method name should also be more explicit about what it does / is for.   Like what if you call this method on a trigger that is in BLOCKED state (and for proper reason is in blocked state)?